src/app/components/PaymentSummary.tsx

```diff
@@ -6,6 +6,30 @@ type Props = {
 };
 
 function PaymentSummary({ amount, amountAlt, description, metadata }: Props) {
+  const metadataElements: JSX.Element[] = [];
+  if (metadata != undefined) {
+    for (const key in metadata) {
+      if (key[0] != "@") {
+        metadataElements.push(
+          <>
+            <dt className="mt-4 font-medium text-gray-800 dark:text-white">
+              {key}
+            </dt>
+            <dd className="mb-0 text-gray-600 dark:text-neutral-500 break-all">
+              {key == "image" ? (
+                <img src={`data:image/png;base64, ${metadata[key]}`} />
+              ) : (
+                metadata[key]
+              )}
+            </dd>
+          </>
+        );
+      }
+    }
+  } else {
+    metadataElements.push(<p>&quot;No Metadata Present&quot;</p>);
```

Comment:
I think at some point we can leave this and just show nothing. 
this does not really give much value to the user.

Reply:
sure!

Reply:
Resolved in this pull request::  https://github.com/getAlby/lightning-browser-extension/pull/1112